### ISSUE TITLE
Can the ip address be added to the Overview page?

### ISSUE THREAD

**Issue by mrpocketsgb** [MicrosoftDocs/Windows-Admin-Center-Ideas-and-Feedback#113]
**Is your feature request related to a problem? Please describe.**
Would be really useful to see the ip address(es) on the Overview page rather than having to click on network.

**Describe the solution you'd like**
As above.

**Describe alternatives you've considered**
As above.

**Additional context**
As above.

**Comment by FDFessler** [comment]
Agree.  Azure displays the IP in the Overview (for what it's worth).

**Comment by soyfrien** [comment]
I think this is already possible by choosing your desired columns. Might be through a context-menu or a button.

**Comment by FDFessler** [comment]
Hi Louis, yes it can be added on the *Inventory page of a Hypervisor Host
or Cluster* using the "|||" column picker, but does not currently show up
on the Overview page of an individual VM or stand-alone machine.  If it is
possible, I haven't found out how to do it.  ---  Freeman
